src/components/GoalCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import Strings from '../../i18n/en';
+import ProgressBar from '../ProgressBar';
+import { GoalCardStyles } from './GoalCardStyle';
+
+export type GoalCardProps = {
+  title: String;
+  taskAssignee: String;
+  taskDescription: String;
+  progress: number;
+};
+
+function GoalCard(props: GoalCardProps) {
+  const styles = GoalCardStyles;
```

Comment:
We are importing the GoalCardStyles , so just use that if not then can also change it with importing GoalCardStyles as styles.